dbusadaptor.cpp

```diff
@@ -22,19 +21,27 @@
 #include <QtQuick/QQuickItem>
 
 DBusAdaptor::DBusAdaptor(QQuickView *view)
-    : QDBusAbstractAdaptor(view), m_view{view}
+    : QDBusAbstractAdaptor(view)
+    , m_view(view)
 {
 }
 
 DBusAdaptor::~DBusAdaptor()
 {
 }
 
-void DBusAdaptor::openFile(const QStringList &files)
+void DBusAdaptor::Open(const QStringList &uris, const QVariantMap &platformData)
 {
-    if (files.count() > 0) {
-        QMetaObject::invokeMethod(m_view->rootObject(), "openFile", Q_ARG(QVariant, files.at(0)));
+    Q_UNUSED(platformData)
+    if (!uris.isEmpty()) {
+        QMetaObject::invokeMethod(m_view->rootObject(), "openFile", Q_ARG(QVariant, uris.at(0)));
     } else {
-        QMetaObject::invokeMethod(m_view->rootObject(), "activate");
+        Activate(platformData);
```

Comment:
As platformData is used / passed onwards from here, it is not really Q_UNUSED, or?

Reply:
It was forgotten there. Originally these had separate implementations but I thought that calling `Activate` would make sense here. I'll remove that `Q_UNUSED` from this function. Well spotted. :+1: